src/modules/CreateChannel/components/InviteUsers/index.tsx

```diff
@@ -76,11 +69,11 @@ const InviteUsers: React.FC<InviteUsersProps> = ({
     }
   }, []);
 
-  // https://stackoverflow.com/a/70302463
-  // https://css-tricks.com/the-trick-to-viewport-units-on-mobile/#css-custom-properties-the-trick-to-correct-sizing
-  // to fix navbar break in mobile
+  // To fix navbar break in mobile we set dynamic height to the scrollable area
   useEffect(() => {
-    appHeight();
+    const appHeight = () => {
```

Comment:
Just minore: let's change this name to something other than `appHeight` which is a state.

Reply:
Yeah sounds good https://github.com/sendbird/sendbird-uikit-react/pull/899/commits/3a8b7ea7f9802ba5c7091294d89c4cbc0fe971b1